client/my-sites/stats/stats-purchase/styles.scss

```diff
@@ -182,6 +183,23 @@ $button-padding: 8px 32px;
 		}
 	}
 
+	.stats-purchase-wizard__card-inner--single {
+		padding: 64px;
+		box-sizing: border-box;
+		flex: 2 2 auto;
+
+		p {
+			width: 36em;
+			@media (max-width: $break-medium) {
+				width: unset;
+			}
+		}
+
+		@media (max-width: $break-medium) {
+			padding: 32px 24px;
+		}
+	}
+
 	.stats-purchase-wizard__card-inner--left {
```

Comment:
Maybe need double check, but the class doesn't seem to be used anywhere anymore.